matlab_simulation/code v1.0/ME597-8-NLP/constraints.m

```diff
@@ -22,3 +22,7 @@
         end
     end
 end
+
+% Velocity constraints
```

Comment:
These are variable bound constraints, I believe, and should be applied at every time step.